.github/workflows/docker.yml

```diff
@@ -0,0 +1,52 @@
+name: Docker
+
+on:
+  schedule:
+    - cron: '20 7 * * *'
```

Comment:
When I look at the last 3 years, I often see times when there were no changes to the repo for several months. That causes that the image has no security updates in this time frames.

So daily is probably too often, but once a week?

Reply:
I agree, there should be more regular builds. I never understood why that was not the case.

Reply:
Yeah, weekly builds would be preferable in my opinion.

Reply:
I don't quite see the need for fequent security updates, since the container should never be exposed to the network. But if it makes you folks happy, let's rebuild weekly.